web/Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM node:16.17-alpine3.15
+FROM node:18-alpine3.18
 
 # Install Chromium for automated testing.
 RUN apk add --no-cache chromium
 ENV CHROME_BIN=/usr/bin/chromium-browser
+ENV NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
Temporary workaround to get the build working. However, we need to update dependencies in package.json. To get the correct fixes.

Reply:
@A808597949 @Ambition-J for awareness